### PR TITLE
NodeSingle route and linking fixes

### PR DESCRIPTION
**Pull request by vinnyjth**
**IMPORTANT: Unless you are doing a release, you should be pointing your PR at the `develop` branch. `master` should not rely on code not published in `npm`**

To test, link with `apps` and then visit the home feed. Tap on the "View More" and a content item should open up. This item opening is the NodeSingle, rather than the usual ContentSingle.

### CODE REVIEW

File: apolloschurchapp/scripts/link-packages.js [hunk 1]
```diff
@@ -46,6 +46,16 @@ dependencies.forEach((dep) => {
       `printf 'y\n' | yarn wml add ${apollosAppsLocation}/packages/apollos-${folderName} ${__dirname}/../node_modules/${dep}`,
       { encoding: 'utf8' }
     );
+    const watchmanConfig = JSON.parse(
+      fs.readFileSync(
+        `${apollosAppsLocation}/packages/apollos-${folderName}/.watchmanconfig`
+      )
+    );
+    watchmanConfig.ignore_dirs = [...watchmanConfig.ignore_dirs, 'coverage'];
```
Comment (vinnyjth) [inline apolloschurchapp/scripts/link-packages.js:54]: Now `wml` won't copy over the coverage folder (which was like 10,000 html files)

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "start": "yarn watch:client & lerna run start --parallel",
     "watch:client": "(cd apolloschurchapp && yarn wml start)",
-    "link-packages": "watchman watch-del-all && lerna run link-packages --parallel",
+    "link-packages": "watchman watch-del-all && lerna run link-packages --parallel && (cd apollos-church-api && yalc update)",
```
Comment (vinnyjth) [inline package.json:6]: That `yalc update` at the end seems to fix a persistent package linking issue

File: apolloschurchapp/src/node-single/NodeSingle.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from 'react';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+
+import { ThemeMixin } from '@apollosproject/ui-kit';
+
+// import { TrackEventWhenLoaded } from '@apollosproject/ui-analytics';
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/NodeSingle.js:7]:
```suggestion
```

File: apolloschurchapp/src/node-single/NodeSingle.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+import {
+  InteractWhenLoadedConnected,
+  NodeSingleConnected,
+} from '@apollosproject/ui-connected';
+
+// import ActionContainer from './ActionContainer';
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/NodeSingle.js:13]:
```suggestion
```

File: apolloschurchapp/src/node-single/NodeSingle.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+import NavigationHeader from '../ui/NavigationHeader';
+
+class ContentSingle extends PureComponent {
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/NodeSingle.js:17]:
```suggestion
class NodeSingle extends PureComponent {
```

File: apolloschurchapp/src/node-single/NodeSingle.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+      >
+        <InteractWhenLoadedConnected nodeId={this.nodeId} action={'COMPLETE'} />
+        {/* <TrackEventWhenLoaded */}
+        {/*   isLoading={loading} */}
+        {/*   eventName={'View Node'} */}
+        {/*   properties={{ */}
+        {/*     title: content.title, */}
+        {/*     itemId: this.itemId, */}
+        {/*   }} */}
+        {/* /> */}
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/NodeSingle.js:51]:
```suggestion
```

File: apolloschurchapp/src/node-single/NodeSingle.js [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+        <NodeSingleConnected nodeId={this.nodeId} />
+        {/* <ActionContainer itemId={id} /> */}
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/NodeSingle.js:53]:
```suggestion
```

File: apolloschurchapp/src/node-single/NodeSingle.js [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+      </ThemeMixin>
+    );
+  }
+}
+
+export default ContentSingle;
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/NodeSingle.js:59]:
```suggestion
export default NodeSingle;
```

File: apolloschurchapp/src/node-single/index.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import { createStackNavigator } from 'react-navigation';
+
+import NodeSingle from './NodeSingle';
+
+const ContentSingleNavigator = createStackNavigator(
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/index.js:5]:
```suggestion
const NodeSingleNavigator = createStackNavigator(
```

File: apolloschurchapp/src/node-single/index.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  {
+    initialRouteName: 'NodeSingle',
+    headerMode: 'float',
+    headerTransitionPreset: 'fade-in-place',
+    navigationOptions: { header: null },
+  }
+);
+
+export default ContentSingleNavigator;
```
Comment (redreceipt) [inline apolloschurchapp/src/node-single/index.js:17]:
```suggestion
export default NodeSingleNavigator;
```

**Review by redreceipt — CHANGES_REQUESTED** [review]
works great. I think just the naming changes and delete the comments are we're good

**Comment by vinnyjth** [comment]
@redreceipt Github needs to give you a PR Suggestion IDE ;)

**Comment by redreceipt** [comment]
That would be awesome

**Comment by vinnyjth** [comment]
@redreceipt Changes applied!

**Review by redreceipt — APPROVED** [review]

**Pull request merged**